README.md

```diff
@@ -387,6 +388,27 @@ epoxyAdapter.setSpanCount(spanCount);
 layoutManager.setSpanSizeLookup(epoxyAdapter.getSpanSizeLookup());
 ```
 
+## Configuration
+
+Use the `@ModuleEpoxyConfig` annotation to specify module wide configuration options for all models in the module. This is a class annotation which you can put on any class in your module. Using an empty class is a fine pattern.
```

Comment:
nit: "module wide" and "in the module" is redundant. You could omit "in the module" and it would still be clear imo.